scripts/langEditor.php

```diff
@@ -212,6 +254,26 @@ function renderSourceTableLine($index, &$pmfLangSource, &$pmfLangTarget, &$langC
 						output += key + " = " + value + ";\n";
 					})
 					textarea.value = output;
+
+					//sending an ajax request with the output to check for possible sintax errors and typos
+					var xhttp = new XMLHttpRequest();
```

Comment:
We can use the Fetch API here instead of XMLHttpRequest. See https://developer.mozilla.org/de/docs/Web/API/Fetch_API

Reply:
Thanks for the patience. I feel stuck on time, using jurassic js.